components/brave_wallet/browser/brave_wallet_utils.cc

```diff
@@ -76,6 +76,26 @@ bool GetUseStagingInfuraEndpoint() {
   return env->HasVar("BRAVE_INFURA_STAGING");
 }
 
+const brave_wallet::mojom::EthereumChain kFilecoinNetworks[] = {
```

Comment:
don't use EthereumChain for Filecoin
do it like this introducing a new `FilecoinNetwork`
https://github.com/brave/brave-browser/issues/20831